doc-content/enterprise-only/processes/filtering-process-instances-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='filtering-process-instances-proc']
+= Filtering process instances using basic or advanced filters
+
+{Central} now provides you with basic and advanced filters to help you filter and search through running process instances. You can filter processes by attributes such as *State*, *Errors*, *Filter By*, *Name*, *Start Date*, and *Last update*. You can also create custom filters using the *Advanced Filters* option. The newly created custom filter is added to the *Saved Filters* pane, which is accessible by clicking on the star icon on the left of the *Manage Process Instances* page.
```

Comment:
In the first sentence above:
`...to help you filter and search through running process instances.`
Filtering can search through process instances no matter their state, it's not limited to running process instances only.

Reply:
Changed as suggested.